flik/hessian/__init__.py

```diff
@@ -0,0 +1,79 @@
+# An experimental local optimization package
+# Copyright (C) 2018 Ayers Lab <ayers@mcmaster.ca>.
+#
+# This file is part of Flik.
+#
+# Flik is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# Flik is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>
+
+
+r"""Hessian classes."""
+
+
+from flik.jacobian.base import Jacobian
+from flik.jacobian.finitediff import ForwardDiffJacobian
+from flik.jacobian.finitediff import CentralDiffJacobian
+
+
+__all__ = [
+    "Hessian",
+    "ForwardDiffHessian",
+    "CentralDiffHessian",
+    ]
+
+
+class Hessian(Jacobian):
```

Comment:
All Hessians are the Jacobian of their corresponding gradient function.  Not all Jacobians are Hessians.